code-challenges.test.js

```diff
@@ -50,11 +109,46 @@ const studyMinutesWeek2 = {
 // Expected output: [10, 15, 20, 45, 60, 65, 100]
 
 // b) Create the function that makes the test pass.
+// create a function "array1" that will  take in an array, and return a new array with the index placed in order
+// create a new but empty array for the new return which will be represented by arr1
+// utilize a series of methods such as push and sort
+
+const array1 = (value) => {
+  const arr1 = [];
+  for (const key in value) {
+    arr1.push(value[key]);
+  }
+  return arr1.sort((a, b) {
+    return a - b;
+  });
+};
+
```

Comment:
Excellent!  I can tell you did some research for this one.  I encourage you to include an explanation of the for in loop, just for your own benefit.  Just a couple syntax notes:  make sure you use `let` when assign variables to an empty array (or any item that may be modified later) and you forgot the => in your sort method.  Otherwise, Fantastic job Christian!